src/InfluxDB/Point.php

```diff
@@ -11,44 +11,51 @@
  */
 class Point
 {
+    /**
+     * @var string
+     */
     private $measurement;
+
     /**
      * @var array
      */
     private $tags = array();
+
     /**
      * @var array
      */
     private $fields = array();
+
     /**
      * @var string
      */
-    private $timestamp = null;
+    private $timestamp;
 
     /**
-     * The timestamp is optional.
-     * If you do not specify a timestamp the server’s local timestamp will be used
-     *
-     * @param string $measurement      Name of the measurement
-     * @param float  $value            Value of the measurement
-     * @param array  $tags             Array of tags
-     * @param array  $additionalFields Array of optional fields
-     * @param int    $timestamp        Optional timestamp
+     * The timestamp is optional. If you do not specify a timestamp the server’s
+     * local timestamp will be used
      *
+     * @param  string $measurement
+     * @param  float  $value
+     * @param  array  $tags
+     * @param  array  $additionalFields
+     * @param  null   $timestamp
      * @throws DatabaseException
      */
-    public function __construct($measurement, $value, array $tags = array(), array $additionalFields = array(), $timestamp = null)
-    {
-
+    public function __construct(
+        $measurement,
+        $value,
+        array $tags = array(),
+        array $additionalFields = array(),
+        $timestamp = null
+    ) {
         if (empty($measurement)) {
             throw new DatabaseException('Invalid measurement name provided');
         }
 
         $this->measurement = (string) $measurement;
-        $this->tags = $tags;
-        $this->fields = $additionalFields;
-
-        $this->fields += array('value' => (float) $value);
+        $this->tags        = $tags;
+        $this->fields      = array_merge($additionalFields, array('value' => (float) $value));
```

Comment:
@thecodeassassin well, then we can improve it:

```
$this->fields['value'] = (float) $value;
```

Will do, there's no need for `+= array('value' => (float) $value)` like it was before. Agree?